src/modules/PortAudio/PortAudioWriter.cpp

```diff
@@ -203,12 +240,36 @@ bool PortAudioWriter::open()
 
 bool PortAudioWriter::openStream()
 {
+#ifdef Q_OS_MACOS
+	bool bitPerfectMode = sets().getBool("BitPerfect");
+	if (bitPerfectMode && (outputDevice.isEmpty() || outputDevice == "Default"))
```

Comment:
IIRC default device is always empty/null. Moreover default device is translated on the list. So `outputDevice == "Default"` not needed.